pgx-tests/src/framework.rs

```diff
@@ -379,8 +492,13 @@ fn start_pg(loglines: LogLines) -> eyre::Result<String> {
 
     // add a shutdown hook so we can terminate it when the test framework exits
     add_shutdown_hook(move || unsafe {
-        let message_string =
-            std::ffi::CString::new("Stopping Postgres\n\n".bold().blue().to_string()).unwrap();
+        let message_string = std::ffi::CString::new(
+            format!("Stopping Postgres PID {}\n\n", pgpid)
```

Comment:
should have more symmetry between the two pid markers in a test's output over a test run's lifetime, and probably don't need double newlines as the next entry will probably have its own framing. or maybe I'm wrong on that last bit.
```suggestion
            format!("stopping postgres (pid={pgpid})\n")
```

Reply:
Looks good!

Reply:
Regarding the newline: They were there before I started doing this work, so while we have a chance to improve it we should. This could probably also be colored too if we wanted to get Super Fancy.